tests/input/BINARY_OP.py

```diff
@@ -0,0 +1,4 @@
+def add(a, b):
+    return a + b

```

Comment:
Please test all cases of binary ops, not just addition (that would have caught the issue above)